src/doc.js

```diff
@@ -71,16 +71,30 @@ const setHome = async (param, body, req) => {
   return http.ok
 }
 
+export const oldHome = async () => {
+  const load = async (name) => (await loadFile(c.homePath, name))
+                               ?.Body
+                               ?.toString()
+
+  const menu     = (await load('old_menu.json')) ?? 'null'
+  const config   = (await load('config.json'))   ?? 'null'
+  const style    = (await load('old_style.css')) ?? defaultStyle
+  const linkText = 'null'
```

Comment:
If the whole difference is just in the old page not linking to notion, is it really necessary to have two versions, old and new? Isn't it harmless to show the link?

Reply:
No, that is not the only difference. new page has a button to Notion handbook and with the link text it links to old (which is the same as old doc-viewer)